composer.json

```diff
@@ -20,6 +20,9 @@
     "require": {
         "php": ">=5.3"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^4|^5|^6|^7|^8|^9"
```

Comment:
I personally the these tests locally with PHPUnit 9.5.28 on PHP 8.1. There are a couple of warnings about obsolete syntax in `phpunit.xml.dist` (copied below), but nothing that prevents the test suite from running. I'm assume there's no harm in including "^5|^6|^7|^8" for maximum compatibility.

----

```
  Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 12:
  - Element 'phpunit', attribute 'syntaxCheck': The attribute 'syntaxCheck' is not allowed.

  Line 19:
  - Element 'filter': This element is not expected.

  Test results may not be as expected.
```